src/lib/apm.ts

```diff
@@ -14,7 +14,6 @@ apm
     logLevel: 'off',
     captureBody: 'all',
     errorOnAbortedRequests: false,
-    logUncaughtExceptions: false,
```

Comment:
This config option was removed since it cannot be turned off any more: https://github.com/elastic/apm-agent-nodejs/blob/main/docs/upgrade-to-v4.asciidoc#loguncaughtexceptions